src/server/models/user.js

```diff
@@ -1,9 +1,9 @@
-import thinky from "./thinky";
+import thinky from './thinky';
```

Comment:
I think the changes in this file are lint-churn.  `prettier` really prefers double-quotes here, so at least please make sure you're using the githook in package.json or running `prettier --write` on changed files on your own.

Reply:
Will do it!